v_5_2_0/files/conf/k8s-addons

```diff
@@ -88,29 +88,29 @@ $KUBECTL delete pods -l k8s-app=kube-proxy -n kube-system
 
 {{ if not .DisableCalico -}}
 
-# apply calico
-CALICO_FILE="calico-all.yaml"
+{{ if eq .Cluster.Kubernetes.CloudProvider "aws" -}}
```

Comment:
This is unrelated but @giantswarm/team-celestial how about doing the same for Azure and removing `.DisableCalico` setting?

It would be easier to update calico that way because we'd have all manifests in a single repo.